src/XMakeBuildEngine/Construction/ProjectMetadataElement.cs

```diff
@@ -49,7 +49,14 @@ public string Name
         }
 
         //  Add a new property with the same name here because this attribute should be public for ProjectMetadataElement,
-        //  but internal for ProjectElement, because we don't want it to be settable for arbintrary elements.
+        //  but internal for ProjectElement, because we don't want it to be settable for arbitrary elements.
```

Comment:
nit: is this a comment before the doc comments? should it just be in `<remarks>`?